devfiles/quarkus-commandmode/devfile.yaml

```diff
@@ -0,0 +1,100 @@
+---
+apiVersion: 1.0.0
+metadata:
+  generateName: quarkus-commandemode-
+projects:
+  -
+    name: quarkus-quickstarts
+    source:
+      type: git
+      location: "https://github.com/quarkusio/quarkus-quickstarts.git"
+      sparseCheckoutDir: getting-started-command-mode
```

Comment:
should add a tag or a branch as well (because sometimes upstream repo is changing and then it stop to work)

Reply:
problem is the repo from quarkus is always moving. each release they do a rebase ...

Reply:
you can't use for example https://github.com/quarkusio/quarkus-quickstarts/releases/tag/1.6.1.Final ?
tag shouldn't move

Reply:
I have tested ... but it was broken at the time ... https://github.com/quarkusio/quarkus-quickstarts/pull/629
So let's use the next release tag